docs/spire.md

```diff
@@ -6,13 +6,58 @@ weight: 1660
 -->
 ⚠️ This is a work in progress: SPIRE support is not yet functional
 
-TaskRun result attestations is currently an alpha experimental feature. Currently all that is implemented is support for configuring Tekton to connect to SPIRE. See TEP-0089 for details on the overall design and feature set.
+TaskRun result attestations is currently an alpha experimental feature. Currently all that is implemented is support for configuring Tekton to connect to SPIRE and enabling TaskRun to sign and verify the TaskRunStatus. See TEP-0089 for details on the overall design and feature set.
 
 This being a large feature, this will be implemented in the following phases. This document will be updated as we implement new phases.
```

Comment:
should this comment be updated now that we've finished the implementation

Reply:
Agreed, also updated the features completed with "done" status.